### PR TITLE
Fix documentation issue in the stdlib mb package

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: stdlib/mb/src/main/ballerina/mb/simple_topic_subscriber.bal [hunk 1]
```diff
@@ -126,6 +132,7 @@ public type TopicSubscriberActions object {
     # Acknowledges a received message
     #
     # + message - Message to be acknowledged
+    # + return - `error` on failure to ackowledge
```
Comment (Anoukh) [inline stdlib/mb/src/main/ballerina/mb/simple_topic_subscriber.bal]: Minor typo

File: stdlib/mb/src/main/ballerina/mb/simple_durable_topic_subscriber.bal [hunk 2]
```diff
@@ -93,10 +94,12 @@ public type SimpleDurableTopicSubscriber object {
 # + port - Hostname of the Ballerina message broker
 # + clientID - Used to identify the JMS client
 # + virtualHost - Name of the virtual host where the virtual host is a path that acts as a namespace
+# + secureSocket - Configuration for the TLS options to be used
 # + connectionFactoryName - JNDI name of the connection factory
 # + acknowledgementMode - JMS session acknowledgement mode. Legal values are "AUTO_ACKNOWLEDGE",
 #                         "CLIENT_ACKNOWLEDGE", "SESSION_TRANSACTED" and "DUPS_OK_ACKNOWLEDGE"
 # + messageSelector - Message selector condition to filter messages
+# + identifier - the identifier of this SimpleDurableTopicSubscriber
```
Comment (daneshk) [inline stdlib/mb/src/main/ballerina/mb/simple_durable_topic_subscriber.bal]: shall we change `this` to `the`

File: stdlib/mb/src/main/ballerina/mb/simple_queue_receiver.bal [hunk 3]
```diff
@@ -128,14 +133,15 @@ public type QueueReceiverActions object {
...
+    # + return - `error` if the acknowledement fails
     public function acknowledge(Message message) returns error? {
         return helper.acknowledge(message.getJMSMessage());
     }
 
     # Synchronously receive a message from Ballerina message broker
     #
     # + timeoutInMilliSeconds - time to wait until a message is received
-    # + return - Returns a message or nill if the timeout exceededs. Returns an error on broker internal error.
+    # + return - `Message` or nill if the timeout exceededs. Returns an `error` on broker internal error.
```
Comment (daneshk) [inline stdlib/mb/src/main/ballerina/mb/simple_queue_receiver.bal]: Shall we change `nill` to `nil`

File: stdlib/mb/src/main/ballerina/mb/simple_queue_sender.bal [hunk 4]
```diff
@@ -57,7 +57,9 @@ public type SimpleQueueSender object {
 
     }
 
-    # Returns the caller action object of the SimpleQueueSender
+    # gets the caller action object of the `SimpleQueueSender`
```
Comment (daneshk) [inline stdlib/mb/src/main/ballerina/mb/simple_queue_sender.bal]: Start with upper case. `Gets`

File: stdlib/mb/src/main/ballerina/mb/simple_queue_receiver.bal [hunk 5]
```diff
@@ -128,14 +133,15 @@ public type QueueReceiverActions object {
     # Acknowledges a received message
     #
     # + message - message to be acknowledged
+    # + return - `error` if the acknowledement fails
```
Comment (daneshk) [inline stdlib/mb/src/main/ballerina/mb/simple_queue_receiver.bal]: should change to `acknowledgement `

File: stdlib/mb/src/main/ballerina/mb/simple_queue_receiver.bal (hunk 3 above)
Comment (daneshk) [reply stdlib/mb/src/main/ballerina/mb/simple_queue_receiver.bal]: change to `exceeds`

File: stdlib/mb/src/main/ballerina/mb/simple_queue_sender.bal [hunk 6]
```diff
@@ -102,7 +105,7 @@ public type QueueSenderActions object {
 
     # Sends a message to Ballerina message broker
     #
-    # + message - message to be sent to Ballerina message broker
+    # + return - `error` on failure to send
```
Comment (daneshk) [inline stdlib/mb/src/main/ballerina/mb/simple_queue_sender.bal]: Shall the complete the sentence. like `to send the Message`

**Pull request merged**